app/models/validations/sales/soft_validations.rb

```diff
@@ -10,7 +10,7 @@ module Validations::Sales::SoftValidations
   def income1_under_soft_min?
     return false unless ecstat1 && income1 && ALLOWED_INCOME_RANGES[ecstat1]
 
-    income1 < ALLOWED_INCOME_RANGES[ecstat1][:soft_min]
+    income1 < ALLOWED_INCOME_RANGES[ecstat1] [:soft_min]
```

Comment:
I feel like this change shouldn't be here anyway. Shouldn't cause conflicts, but is there a reason this is here?